### ISSUE TITLE
[Bug]: Separate folders / books listed as a single book

### ISSUE THREAD

**Issue by MadBobUK** [advplyr/audiobookshelf#834]
### Describe the issue

List of books in series folders under single author shown as one book with errors

Quite a lot of the books are numbered as the series Undead Part 1, Undead Part 2 etc.

see attached file for file/folder structure

### Steps to reproduce the issue

[Audiobooks.txt](https://github.com/advplyr/audiobookshelf/files/9104489/Audiobooks.txt)

### Audiobookshelf version

audiobookshelf@2.0.23 (Windows 10 Docker)

### How are you running audiobookshelf?

Docker

**Comment by Benni-chan** [comment]
I could reproduce the issue.

If I copy a folder with multiple subfolders all at once, I get one audiobook with multiple files.

If I first create the series folder and then only copy one subfolder with one book and wait until it got imported in ABS and only then copy the remaining books, then I have every book in abs as it should be.

In my case the folder structure is:
\John Scalzi\Old Man's War\John Scalzi - Old Man's War [01] - Old Man's War\Old Man's War 01 - Old Man's War (William Dufris).m4b
\John Scalzi\Old Man's War\John Scalzi - Old Man's War [02] - The Ghost Brigades\Old Man's War 02 - The Ghost Brigades (William Dufris).m4b
and so on...

**Comment by advplyr** [comment]
@Benni-chan I just tried to reproduce this following your method and the books pulled in individually.

To test this I created the folder structure outside of my library first.
`/Author Name/Series Name/Book A/Track.mp3`
`/Author Name/Series Name/Book B/Track.mp3`

Then I copied the `Author Name` folder and pasted it inside my audiobookshelf library folder. The scanner correctly identified both books.

**Comment by advplyr** [comment]
Thanks for the clarification. I have made an update that will be included in the next release.

With this update I tested by creating this folder structure outside of my library first.
`/Author Name/random text file.txt`
`/Author Name/Series Name/random text file.txt`
`/Author Name/Series Name/Book A/Track.mp3`
`/Author Name/Series Name/Book B/Track.mp3`

Adding a text file inside the author and the series folder. Then copied the `Author Name` folder into my library folder. The scanner correctly identified both books.

Important to note that the "book" media type will consider any audio file OR ebook file has a valid media file. Which means had I stored a PDF, EPUB, MOBI, AZW3, CBR or CBZ instead of a TXT file inside the series or author folder then it would treat that entire folder as a single item.
